frontend/amundsen_application/static/js/config/config-utils.ts

```diff
@@ -77,7 +80,42 @@ export function getResourceNotices(
   const { notices } = AppConfig.resourceConfig[resourceType];
 
   if (notices && notices[resourceName]) {
-    return notices[resourceName];
+    const thisNotice = notices[resourceName];
+    if (typeof thisNotice.messageHtml !== 'string') {
```

Comment:
I think it would be better do do a positive check like:
```suggestion
    if (typeof thisNotice.messageHtml === 'function') {
```
I will be more safe and explicit